cmd/clairctl/clair/errors.go

```diff
@@ -0,0 +1,13 @@
+package clair
+
+import "errors"
+
+const oSNotSupportedValue = "worker: OS and/or package manager are not supported"
```

Comment:
We should rename + rephrase it. We'd like not to mention OS/packages anymore as Clair is far more general than that now.